### PR TITLE
Application Appendix Rewrite

### PR DESCRIPTION
**Pull request by MrLongbottom**
Removed most of the text i considered non-essential or focusing on something other than applications.
Also removed most of the ending about how metadata affects topics as we already do mention it in the article and the questions we ask are not answered anywhere so they might end up hurting us.
The new text is focused more on giving more examples of applications and making the examples a bit more concrete.

### CODE REVIEW

File: sections/appendix/applications.tex [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
+News media groups, like Nordjyske, are trying to find new ways of integrating smarter and more intelligent methods for keeping their customers, and using topic modeling can help improve their processes such as : searching, recommendation, grouping of articles, information completion, etc.
+We will briefly give an overview of each of these to explain how topic modelling might play a role in improving these processes.
+
+The first example is to improve search functionality for their articles.
+This problem could be alleviated by using the topics created by our topic models, by performing query expansions finding similar words to the ones that appear in the query, providing more context to find results from.
+Once initial search results have been found, topic models could also be used to find other results with similar topics.
+These techniques can be particularly useful as they can help find articles that do not use any of the words in given query, but are still relevant to the query, which is a property that most basic search algorithms does not possess.
 
 One very important problem, within news agencies, is recommendation of articles.
```
Comment (MrLongbottom) [inline sections/appendix/applications.tex]:
```suggestion
Another very important problem, within news agencies, is recommendation of articles.
```

File: sections/appendix/applications.tex [hunk 2]
```diff
@@ -1,22 +1,23 @@
...
-We want to see how any meta-information can impact the topics created by the \gls{lda}. 
-How can meta-information, such as geographic locations from the Category or Taxonomy fields, influence the topics created?
-Are we able to figure out which types of articles do well in certain regions of a country based on this information? 
-Can time information be used to figure out whether a certain article gets better performance on another date?
+Another very important problem, within news agencies, is recommendation of articles.
+Topic models can be used particularly for content-based filtering, finding other articles that have similar topics either to a users preferences based on their history or to another article.
+
+Grouping together items, or clustering, can serve many different purposes. Topic modelling provides a new way of grouping together articles based on topic similiarity.
+
+A topic model might also be used to fill in missing information to the dataset.
```
Comment (MrLongbottom) [inline sections/appendix/applications.tex]:
```suggestion
A topic model might also be used to fill in missing meta information in a article dataset.

```

File: sections/appendix/applications.tex [hunk 3]
```diff
@@ -1,22 +1,24 @@
...
 This idea of taking meta-information into account seems to be overlooked, even though a few papers have touched upon this area, the application possibilities for these kind of meta-information integrations are endless.
 
-News media groups, like Nordjyske, are trying to find new ways of integrating smarter and more intelligent methods for keeping their customers, and using topic modeling can help improve their processes in different ways.
-One case could be that they want better search functionality for their articles.
-This problem could be alleviated by using the topics created by our \gls{lda} model, to specifically target certain types of topics and group articles together with very similar topic distributions, like sports articles.
+News media groups, like Nordjyske, are trying to find new ways of integrating smarter and more intelligent methods for keeping their customers, and using topic modeling can help improve their processes such as : searching, recommendation, grouping of articles, information completion, etc.
```
Comment (rasmus-e-christensen) [inline sections/appendix/applications.tex]:
```suggestion
News media groups, like Nordjyske, are trying to find new ways of integrating smarter and more intelligent methods for keeping their customers, and using topic modeling can help improve their processes such as searching, recommendation, the grouping of articles, information completion, etc.
```

File: sections/appendix/applications.tex [hunk 4]
```diff
@@ -1,22 +1,24 @@
...
+We will briefly give an overview of each of these to explain how topic modelling might play a role in improving these processes.
```
Comment (rasmus-e-christensen) [inline sections/appendix/applications.tex]:
```suggestion
We will briefly give an overview of each of these to explain how topic modeling might play a role in improving these processes.
```

File: sections/appendix/applications.tex [hunk 5]
```diff
@@ -1,22 +1,24 @@
...
 
-One very important problem, within news agencies, is recommendation of articles.
-Many algorithms have already been developed for solving the recommendation problem but incorporating topics within these models could improve the recommendation if scoring of the articles was possible.
-Then, scoring of the articles could be used in recommendation for a wide variety of different users, who might take a liking to a specific article or a topic which is popular at that moment.
+The first example is to improve search functionality for their articles.
+This problem could be alleviated by using the topics created by our topic models, by performing query expansions finding similar words to the ones that appear in the query, providing more context to find results from.
+Once initial search results have been found, topic models could also be used to find other results with similar topics.
+These techniques can be particularly useful as they can help find articles that do not use any of the words in given query, but are still relevant to the query, which is a property that most basic search algorithms does not possess.
```
Comment (rasmus-e-christensen) [inline sections/appendix/applications.tex]:
```suggestion
These techniques can be particularly useful as they can help find articles that do not use any of the words in a given query but are still relevant to the query, which is a property that most basic search algorithms do not possess.
```

File: sections/appendix/applications.tex [hunk 6]
```diff
@@ -1,22 +1,24 @@
...
 
-One important goal of ours is to emphasize the importance of metadata and how it can affect future applications.
-As \citet{author_topic_2012} described, they investigate how authors can have an influence on the topics created by the \gls{lda}.
-We want to see how any meta-information can impact the topics created by the \gls{lda}. 
-How can meta-information, such as geographic locations from the Category or Taxonomy fields, influence the topics created?
-Are we able to figure out which types of articles do well in certain regions of a country based on this information? 
-Can time information be used to figure out whether a certain article gets better performance on another date?
+Another very important problem, within news agencies, is recommendation of articles.
```
Comment (rasmus-e-christensen) [inline sections/appendix/applications.tex]:
```suggestion
Another very important problem, within news agencies, is the recommendation of articles.
```

File: sections/appendix/applications.tex [hunk 7]
```diff
@@ -1,22 +1,24 @@
...
+Topic models can be used particularly for content-based filtering, finding other articles that have similar topics either to a users preferences based on their history or to another article.
```
Comment (rasmus-e-christensen) [inline sections/appendix/applications.tex]:
```suggestion
Topic models can be used particularly for content-based filtering, finding other articles with similar topics either to users' preferences based on their history, or other similar articles.
```

File: sections/appendix/applications.tex [hunk 8]
```diff
@@ -1,22 +1,24 @@
...
+
+Grouping together items, or clustering, can serve many different purposes. Topic modelling provides a new way of grouping together articles based on topic similiarity.
```
Comment (rasmus-e-christensen) [inline sections/appendix/applications.tex]:
```suggestion
Grouping together items, or clustering, can serve many different purposes. 
Topic modeling provides a new way of grouping together articles based on topic similarity.
```

File: sections/appendix/applications.tex [hunk 9]
```diff
@@ -1,22 +1,24 @@
...
+
+A topic model might also be used to fill in missing meta information in a article dataset.
```
Comment (rasmus-e-christensen) [inline sections/appendix/applications.tex]:
```suggestion
A topic model might also be used to fill in missing meta information in an article dataset.
```

**Review by rasmus-e-christensen — CHANGES_REQUESTED** [review]
nogle enkelte stavefejl :) eller ser det fint ud

File: sections/appendix/applications.tex [hunk 10]
```diff
@@ -1,22 +1,24 @@
...
+News media groups, like Nordjyske, are trying to find new ways of integrating smarter and more intelligent methods for keeping their customers, and using topic modeling can help improve their processes such as : searching, recommendation, grouping of articles, information completion, etc.
+We will briefly give an overview of each of these to explain how topic modelling might play a role in improving these processes.
 
-One very important problem, within news agencies, is recommendation of articles.
-Many algorithms have already been developed for solving the recommendation problem but incorporating topics within these models could improve the recommendation if scoring of the articles was possible.
-Then, scoring of the articles could be used in recommendation for a wide variety of different users, who might take a liking to a specific article or a topic which is popular at that moment.
+The first example is to improve search functionality for their articles.
+This problem could be alleviated by using the topics created by our topic models, by performing query expansions finding similar words to the ones that appear in the query, providing more context to find results from.
```
Comment (Starrock45) [inline sections/appendix/applications.tex]:
```suggestion
This problem could be alleviated by using the topics created by our topic models for performing query expansions finding similar words to the ones that appear in the query, providing more context to find results from.
```

File: sections/appendix/applications.tex (hunk 6 above)
Comment (Starrock45) [inline sections/appendix/applications.tex]:
```suggestion
Another very important problem within news agencies is the recommendation of articles.
```

File: sections/appendix/applications.tex [hunk 11]
```diff
@@ -1,22 +1,24 @@
...
-Are we able to figure out which types of articles do well in certain regions of a country based on this information? 
-Can time information be used to figure out whether a certain article gets better performance on another date?
+Another very important problem, within news agencies, is recommendation of articles.
+Topic models can be used particularly for content-based filtering, finding other articles that have similar topics either to a users preferences based on their history or to another article.
+
+Grouping together items, or clustering, can serve many different purposes. Topic modelling provides a new way of grouping together articles based on topic similiarity.
+
+A topic model might also be used to fill in missing meta information in a article dataset.
+
+For example with the taxonomy-topic model, we sample new taxonomies for the majority of the dataset which does not already have a taxonomy entry.
```
Comment (Starrock45) [inline sections/appendix/applications.tex]:
```suggestion
For example, with the taxonomy-topic model, we sample new taxonomies for the majority of the dataset, which does not already have a taxonomy entry.
```

**Review by Starrock45 — APPROVED** [review]
Kun enkelte ting.

**Review by rasmus-e-christensen — APPROVED** [review]

**Pull request merged**